ethcore/trace/src/types/trace.rs

```diff
@@ -113,6 +242,9 @@ pub struct Create {
 	pub gas: U256,
 	/// The init code.
 	pub init: Bytes,
+	/// Creation method (CREATE vs CREATE2).
```

Comment:
I think we should add a not to the docs for this type that data of this type ends up in the database and so great care is necessary when making changes.